app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentRow.kt

```diff
@@ -52,9 +52,9 @@ data class OverdueAppointmentRow(
           gender = overdueAppointment.gender,
           age = DateOfBirth.fromOverdueAppointment(overdueAppointment, clock).estimateAge(clock),
           phoneNumber = overdueAppointment.phoneNumber?.number,
-          bpSystolic = overdueAppointment.bloodPressure.systolic,
-          bpDiastolic = overdueAppointment.bloodPressure.diastolic,
-          bpDaysAgo = calculateDaysAgoFromInstant(overdueAppointment.bloodPressure.recordedAt, clock),
+          bpSystolic = overdueAppointment.bloodPressure?.systolic ?: 0,
```

Comment:
I don't think we should use an elvis operator here, wdyt?

Reply:
We'll keep this PR from being merged, until the PR on showing last seen gets merged.